client/src/component/Register.js

```diff
@@ -72,6 +73,48 @@ const Register = () => {
       name:"Upi number"
     }
   ]
+  const navigate = useNavigate();
+  const [details,setDetails] =useState({tournament_name:"",organiser_name:"",entry_fees:"",email:"",contact:"",start_date:"",state:"",city:"",pincode:"",upiQr:"",upinumber:""})
+  let name, value;
+  const handelInputs = (e) =>{
+        console.log(e)
+        name= e.target.name;
+        value = e.target.value;
+        setDetails({...details, [name]:value});
+        
+   }
+   const PostData = async(e) =>{
+    e.preventDefault();
+    const {tournament_name,organiser_name,entry_fees,email,contact,start_date,state,city,pincode,upiQr,upinumber} = details;
+  const res = await fetch(`${process.env.REACT_APP_API}/create`,{
+      method:"POST",
+      headers:{
+          "Content-Type" : "application/json"
+      },
+      body: JSON.stringify({
+        tournament_name,organiser_name,entry_fees,email,contact,start_date,state,city,pincode,upiQr,upinumber
+      })
+  });
+ 
+  const tdata = await res.json();
+  // console.log(data);
+  // if(data.error){
+  //     toast.error(data.error)
+
+  // }
+  // else{
+      if(tdata.status===422 || !tdata ){
+          window.alert("Try again");
+          console.log("invalid Registration")
+        }
+        else{
+          window.alert("Tournament created sucessfully");
```

Comment:
Instead of alert you can explore react toastify. (https://www.npmjs.com/package/react-toastify)